internal/suites/example/compose/authelia/resources/entrypoint-frontend.sh

```
@@ -1,5 +1,9 @@
 #!/bin/sh
 
-set -x
+PNPM_MODULE="./node_modules/.modules.yaml"
 
-pnpm install --force --frozen-lockfile && pnpm start
+if [[ -f ${PNPM_MODULE} ]]; then
```

Shellcheck

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)